# Constants
uint8 NAV_IDLE = 0
uint8 NAV_NAVIGATING = 1

uint8 status
geometry_msgs/Pose2D[] currentPath
